packages/wonder-blocks-data/src/components/__tests__/data.test.tsx

```diff
@@ -54,7 +54,6 @@ describe("Data", () => {
                 // Act
                 render(
                     <Data handler={fakeHandler} requestId="ID">
-                        {/* @ts-expect-error [FEI-5019] - TS2322 - Type 'Mock<null, [], any>' is not assignable to type '((result: Result<TData>) => ReactElement<any, string | JSXElementConstructor<any>>) & ReactNode'. */}
```

Comment:
**question[no-change-required]:** Is there a way to suppress this for a whole file?

Reply:
**suggestion:** Could just set the `fakeChildrenFn` value to `: any` to get around this. That seems like the easy fix.

Reply:
This change removes a bunch of suppressions.  There are still a bunch of suppressions like:
```
// @ts-expect-error [FEI-5019] - TS2769 - No overload matches this call.
                await act(() => response1);
```
We could look at tweaking the types for `act` in the future.

Reply:
It's likely we just need to tweak the variables, like `response1` to be a compatible type like `any`.